apps/vx-scan/backend/test/helpers/app_helpers.ts

```diff
@@ -40,38 +47,63 @@ import { createWorkspace, Workspace } from '../../src/util/workspace';
 // @ts-ignore
 global.fetch = fetch;
 
-export function postTemplate(
-  app: Application,
-  path: string,
-  ballot: BallotPackageEntry
-): request.Test {
-  return request(app)
-    .post(path)
-    .accept('application/json')
-    .attach('ballots', Buffer.from(ballot.pdf), {
-      filename: ballot.ballotConfig.filename,
-      contentType: 'application/pdf',
-    })
-    .attach(
-      'metadatas',
-      Buffer.from(
-        new TextEncoder().encode(JSON.stringify(ballot.ballotConfig))
-      ),
-      { filename: 'ballot-config.json', contentType: 'application/json' }
-    )
-    .attach(
-      'layouts',
-      Buffer.from(new TextEncoder().encode(JSON.stringify(ballot.layout))),
-      {
-        filename: ballot.ballotConfig.layoutFilename,
-        contentType: 'application/json',
+type MockFileTree = MockFile | MockDirectory;
+type MockFile = Buffer;
+interface MockDirectory {
+  [name: string]: MockFileTree;
+}
+
+function writeMockFileTree(destinationPath: string, tree: MockFileTree): void {
+  if (Buffer.isBuffer(tree)) {
+    // Sleep 1ms to ensure that each file is created with a distinct timestamp
+    execSync('sleep 0.01');
```

Comment:
Seems like this should be in the loop, maybe?

Reply:
Would the goal be to make sure it also creates directories with a unique timestamp? I don't need that property for the current usage, but seems reasonably defensive, so I will change.